zeppelin-web/package-lock.json

```diff
@@ -154,17 +154,19 @@
       "resolved": "https://registry.npmjs.org/amdefine/-/amdefine-1.0.1.tgz",
       "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU="
     },
-    "angular": {
```

Comment:
What is this change ?

Reply:
I think that's what he has in his setup - it's all other PRs as well